src/main/java/frc/robot/Robot.java

```diff
@@ -48,34 +48,44 @@ public class Robot extends CommandRobot {
     @Autonomous(defaultAuto = true)
     public CommandBase exampleAuto = auto.oneConeAuto();
 
-    private CommandBase scoreHigh = sequence(
+    private CommandBase driveScoreHigh = sequence(
             armRotate.moveTo(EnumLevel.HIGH_SCORE), drive.driveToNearest(), arm.moveTo(EnumLevel.HIGH_SCORE),
             intake.coneRelease());
 
-    private CommandBase scoreHighNode = sequence(
+    private CommandBase driveScoreHighNode = sequence(
             armRotate.moveTo(EnumLevel.HIGH_SCORE), drive.driveToNearest(),
             new ConditionalCommand(arm.moveTo(EnumLevel.HIGH_SCORE), runOnce(() -> {
             }), () -> {
                 return gamePieceSub.get() == "Cone";
             }));
 
-    public CommandBase scoreMidNode = sequence(
+    public CommandBase driveScoreMidNode = sequence(
             armRotate.moveTo(EnumLevel.MEDIUM_SCORE), drive.driveToNearest(),
             new ConditionalCommand(arm.moveTo(EnumLevel.MEDIUM_SCORE), runOnce(() -> {
             }), () -> {
                 return gamePieceSub.get() == "Cone";
             }));
 
+    public CommandBase scoreMidNode = sequence(
+            armRotate.moveTo(EnumLevel.MEDIUM_SCORE), (arm.moveTo(EnumLevel.MEDIUM_SCORE)),
+            (armRotate.moveTo(EnumLevel.MEDIUM_SCORE_ACTUAL)));
+
+    public CommandBase scoreHighNode = sequence(
+            armRotate.moveTo(EnumLevel.HIGH_SCORE), (arm.moveTo(EnumLevel.HIGH_SCORE)),
+            (armRotate.moveTo(EnumLevel.HIGH_SCORE_ACTUAL)));
```

Comment:
Shouldn't this sequence also retract the arm?

Reply:
Yes it most certainly should. I thought that was in there but apparently not, I'll put it in.